runtime/libffi/z/ffi64.c

```diff
@@ -919,6 +921,16 @@ ffi_prep_cif_machdep(ffi_cif *cif)
       /* Check how a structure type is passed.  */
       if (type == FFI_TYPE_STRUCT)
 	{
+		/* ffi_check_struct_type() will return FFI_TYPE_UINT64 for a structure with
+			exactly two floating point values of short format which is still OK as
+			in calculation for how many bytes needed in the stack, it will count GPR
+			for such parameters and one double word per argument will be requested.
+			For a structure with two long format floating point values, it will set type to
+			POINTER and allocate two doublewords which is correct need.
+			TODO: There is definitely lot of duplication when it comes to calculating
+			number of bytes needed for parameters, and it will overestimate how much
+			space it needs. Update this routine to correctly identify how many
+			GPRs/FPRs and overflow parameters are needed and set bytes accordingly. */
 		type = ffi_check_struct_type (*ptr);
```

Comment:
The indentation still does not match the rest of this file.
To be more specific, line 924 should start with a tab, two spaces, then the `/`. The following lines should have three extra spaces. Lines 934 and 936 should be indented the same as line 924.

Line 943 should begin with six spaces, line 944 and 949 with one tab, etc.

Reply:
Formatting on the file is bit messed up causing my editor to mess-up. I do see inconsistencies throughout the file. I have tried to address comments in https://github.com/eclipse-openj9/openj9/compare/2bd00bfe772af4a4d85446a336a0ddd079b94090..d2f034b4f4f8c0329a7eba8d393ea888518b8bfc